pygeo/DVGeometryVSP.py

```diff
@@ -370,7 +431,14 @@ def pointSetUpToDate(self, ptSetName):
             return True
```

Comment:
That is just how its done across pygeo. See: https://github.com/mdolab/pygeo/blob/master/pygeo/DVGeometry.py#L1393-L1411 When a `pointSet` is added, an entry to the `self.updated` dictionary is also added. I believe that if check is to avoid the code raising a key error if the entry is not in the dictionary. I don't know if this regularly happens in the code; i.e. if some higher up code checks if a pointset is up to date even though the pointset is not tracked by that dvgeo.